lib/index.js

```diff
@@ -24,9 +27,27 @@ mercutio.middleware = function({ resolveRoles = getRolesFromAuthorizationHeader,
 		req.identity.demand = (role) => {
 			if (!roles.isAny(role)) onDemandFail(req, res, next);
 		};
-
-		return next();
-	};
+		req.identity = { user: null, authenticated: false };
+		let token = req.cookies.Authentication;
+		if (!token) { // header as a fallback
+			token = req.header(header);
+		}
```

Comment:
Unify the above lines. What about:

`let token = req.cookies.Authentication || req.header(header);`

Double `if` makes the code harder to read.